examples/ibm-key-protect/README.md

```diff
@@ -21,6 +17,12 @@ Terraform 0.12. Pin module version to `~> v1.4.0`. Branch - `master`.
 
 Terraform 0.11. Pin module version to `~> v0.25.0`. Branch - `terraform_v0.11.x`.
 
+## Deprecation Notice
+
+  The resource `ibm_kp_key` is deprecated and replaced with `ibm_kms_key`. 
```

Comment:
Might be helpful to say when the deprecation started/took effect.

Reply:
unfortunately, i dont know. But by my guesses it was over a year ago to 2 years ago